test/create-article.ts

```diff
@@ -1,10 +1,10 @@
 import { blankNode, literal, quad } from '@rdfjs/data-model';
-import { dataset } from '@rdfjs/dataset';
 import { DatasetCore, Quad_Subject as QuadSubject } from 'rdf-js';
+import datasetFactory from '../src/dataset-factory';
```

Comment:
Not sure why destructuring no longer worked.

Reply:
Did the export change to the `export = X` form? It is not possible to destructure commonjs-style imports when importing with ES module

Reply:
Nope: https://github.com/libero/article-store/blob/2e7704d027f585282e51c66b6f4e7acf3c7803cc/src/dataset-factory.ts#L4-L6

Reply:
Ah, the same is indeed true for default export. This is not in fact object destructuring, despite similar appearance. Cue [this SO answer](https://stackoverflow.com/a/43987935)

Reply:
Aha, good find.

Anyhow, ignoring as https://github.com/rdf-ext/rdf-dataset-indexed/pull/3#discussion_r359231278 will remove the need for this layer.